src/js/angular/core/directives/queryeditor/templates/query-editor.html

```diff
@@ -76,19 +76,15 @@
                                 ng-show="!notoolbarSaved && isUser()"
                                 ng-click="saveTab(currentQuery.id); saveQuery(currentQuery);"
                                 data-container="#queryEditor"
-                                uib-tooltip="{{'core.create.saved.query' | translate}}"
-                                tooltip-append-to-body="true"
-                                tooltip-trigger="mouseenter">
```

Comment:
How you handle mouseenter? Is it natively handled by the new lib?

Reply:
It is handled by the directive by adding event listeners for `mouseenter` and `mouseleave` events. Those are default, but I have made it possible to set custom show/hide trigger to be used if needed.